objects/official/water/rct2.dlc.wtrpink.json

```diff
@@ -0,0 +1,378 @@
+                    "#B72C94",
+                    "#A62B87",
+                    "#962A7B",
+                    "#85296E",
+                    "#85296E",
+                    "#85296E",
+                    "#85296E",
+                    "#85296E",
+                    "#85296E",
+                    "#85296E",
+                    "#85296E",
+                    "#982B7D",
+                    "#AB2D8C",
+                    "#BF309B"
+                ]
+            },
+            "waves-2": {
+                "index": 48,
+                "colours": [
+                    "#A62385",
+                    "#98237A",
+                    "#89226F",
+                    "#7A2164",
+                    "#6B2058",
+                    "#6B2058",
+                    "#6B2058",
+                    "#6B2058",
+                    "#6B2058",
+                    "#6B2058",
+                    "#6B2058",
+                    "#6B2058",
+                    "#7B2164",
+                    "#8B2271",
+                    "#9C247E"
+                ]
+            },
+            "sparkles-0": {
+                "index": 80,
+                "colours": [
+                    "#B22477",
+                    "#C22C8B",
+                    "#D2349F",
+                    "#E23CB3",
+                    "#F244C7",
+                    "#F244C7",
+                    "#F244C7",
+                    "#F244C7",
+                    "#F244C7",
+                    "#F244C7",
+                    "#F244C7",
+                    "#F244C7",
+                    "#DB3AAC",
+                    "#C43091",
+                    "#AD2675"
+                ]
+            },
+            "sparkles-1": {
+                "index": 96,
+                "colours": [
+                    "#9E2069",
+                    "#A92578",
+                    "#B52B88",
+                    "#C03197",
+                    "#CC37A7",
+                    "#CC37A7",
+                    "#CC37A7",
+                    "#CC37A7",
+                    "#CC37A7",
+                    "#CC37A7",
+                    "#CC37A7",
+                    "#CC37A7",
+                    "#BA2F91",
+                    "#A8277B",
+                    "#961E64"
+                ]
+            },
+            "sparkles-2": {
+                "index": 112,
+                "colours": [
+                    "#851B58",
+                    "#8B1F62",
+                    "#91236C",
+                    "#972776",
+                    "#9E2B81",
+                    "#9E2B81",
+                    "#9E2B81",
+                    "#9E2B81",
+                    "#9E2B81",
+                    "#9E2B81",
+                    "#9E2B81",
+                    "#9E2B81",
+                    "#962674",
+                    "#8E2166",
+                    "#851B58"
+                ]
+            }
+        }
+    },
+    "strings": {
+        "name": {
+            "en-GB": "Pink Water"
```

Comment:
Add a Dutch translation: `nl-NL: "Roze water"`
(don't forget to add a trailing comma to the `en-GB` entry as well)